assets/js/modules/adsense/setup/adsense-setup.js

```diff
@@ -61,7 +61,7 @@ class AdSenseSetupWidget extends Component {
 
 	async getAccounts() {
 		try {
-			const responseData = await data.get( TYPE_MODULES, 'adsense', 'accounts' );
+			const responseData = await data.get( TYPE_MODULES, 'adsense', 'accounts', { maybeSetAccount: true } );
```

Comment:
What's the reason for this change here? I didn't see it mentioned in the PR and it's a bit of a confusing param name 😅

Reply:
@tofumatt I mentioned in the "Relevant technical choices" section. The API route in PHP was running some ugly side effect logic, which I've now wrapped into this confusing name parameter (confusing intentionally because it shouldn't be used). I've added it to all existing code calling that route so that the behavior doesn't change, while preventing that side effect to run for our refactored code.